ServerCore/Pages/Puzzles/Play.cshtml

```diff
@@ -49,6 +49,34 @@
     </div>
 }
 
+@if (Model.Event.IsAlphaTestingEvent && Model.Event.EmbedPuzzles && Model.VisibleTeamPuzzleViews.Any())
```

Comment:
I keep forgetting, but what happens when it is not an embeded puzzle? Will alpha testing not work?

Reply:
Alpha testing currently won't work without embedding because without embedding there's no way to grey out the puzzle. If someone calls for a non-embedded event with alpha testing, we'll decide then what that really means.